.run/build.sh

```diff
@@ -0,0 +1,11 @@
+ROOT_DIR=/git/datadog-agent
+
+if [[ -z $DD_SKIP_VENV ]]; then
+DD_VENV_DIR=venv
+
+source "$DD_VENV_DIR"/bin/activate
```

Comment:
```suggestion
  DD_VENV_DIR=venv
  source "$DD_VENV_DIR"/bin/activate
```

Reply:
This is irrelevant now as I removed the ability to run the script without a venv